inmem-journal/src/main/scala/nl/pragmasoft/pekko/persistence/inmemory/extension/InMemoryJournalStorage.scala

```diff
@@ -101,7 +102,11 @@ class InMemoryJournalStorage(serialization: Serialization) extends Actor with Ac
   }
 
   def writelist(ref: ActorRef, xs: Seq[JournalEntry]): Unit = {
-    val ys = xs.map(_.copy(ordering = incrementAndGet)).groupBy(_.persistenceId)
+    val ys = xs.map(_.copy(ordering = incrementAndGet))
+      .groupBy(_.persistenceId)
+      .view
+      .mapValues(_.toVector)
+      .toMap
```

Comment:
this is probably efficient but I had to map the result to get a type that matched the existing map